src/screens/DiscoveryScreenV2/elements/RecentSearchItem.js

```diff
@@ -1,55 +1,51 @@
 import * as React from 'react';
-import {Pressable, StyleSheet, Text} from 'react-native';
+import { Pressable, StyleSheet, Text } from 'react-native';
 
 import IcArrowUpRight from '../../../assets/icons/ic_arrow_up_right';
-import {fonts} from '../../../utils/fonts';
-import {COLORS} from '../../../utils/theme';
+import { colors } from '../../../utils/colors';
+import { fonts } from '../../../utils/fonts';
 
 /**
  * @typedef {Object} RecentSearchItemsProp
  * @property {String} text
  * @property {Function} onItemClicked
  */
 /**
- *
+ * 
  * @param {RecentSearchItemsProp} prop
  */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The JSDoc comments for the `RecentSearchItems` component are missing a description. It's good practice to include a brief summary of what the component does.
```diff
  /**
+  * RecentSearchItems is a component that renders a clickable item with text and an arrow icon.
   * @typedef {Object} RecentSearchItemsProp
   * @property {String} text
   * @property {Function} onItemClicked
   */
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->